src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/GetAzureRmDataLakeStoreChildItem.cs

```diff
@@ -29,7 +29,7 @@ public class GetAzureDataLakeStoreChildItem : DataLakeStoreFileSystemCmdletBase
         [Alias("AccountName")]
         public string Account { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = true,
```

Comment:
this is a breaking change, is this in the migration guide?

Reply:
@markcowl If you see the original cmdlet, both Account and Path has the same position which is wrong. So I am correcting this. I have run this by @cormacpayne  and he said that since two parameters can't be same poistion this change is fine. Let me know if you want to undo the change.